babelsberg/constraintinterpreter.js

```diff
@@ -1099,8 +1154,8 @@ Object.subclass('ConstrainedVariable', {
                     }
             }
 
-            if (this._constraints.length > 0 && !hasEnabledConstraint)
-                return;
+            // if (this._constraints.length > 0 && !hasEnabledConstraint)
+            //     return;
```

Comment:
Actually, these lines must reappear for the reevaluation, anyway. There is a big BUG comment in a test in #25 to point this out.